app/config/Config.scala

```diff
@@ -27,8 +27,6 @@ class Config(config: Configuration) extends AwsInstanceTags {
   val pandaAuthCallback = getConfigString("panda.authCallback")
   val pandaSystem = getConfigString("panda.system")
 
-  val dbConfig = config.underlying.getConfig("ctx")
```

Comment:
Yeah I don't understand how it came back I think that I ran into issues with a tricky merge...